src/components/m-accordion-item/_example.html

```diff
@@ -0,0 +1 @@
+<!--Please create here a HTML exmple by using just default HTML tags-->
```

Comment:
Newline missing.

@LucaMele This is probably missing in `new` scaffolding script.